README.md

```diff
@@ -1,5 +1,35 @@
-# UKHO Docs
+# Software Engineering at the UKHO
 
-Documentation around digital development project, teams, behaviors and processes at the UKHO.
+This repo contains documentation about digital delivery teams, behaviors and processes at the UKHO.
 
-Users can find the documentation at https://docs.ukho.dev
+We have eight software engineering teams that develop and support software using the .Net and Java stacks.  
```

Comment:
Is this limiting - what about JavaScript, the tools used in DE/DS ?

Reply:
Yep, will ammend.